back/app/models/idea.rb

```diff
@@ -220,6 +222,35 @@ def fix_comments_count_on_projects
   def update_phase_ideas_count(_)
     IdeasPhase.counter_culture_fix_counts only: %i[phase]
   end
+
+  def validate_creation_phase
+    return unless creation_phase
+
+    if creation_phase.project_id != project_id
+      errors.add(
+        :creation_phase,
+        :invalid_project,
+        message: 'The creation phase must be a phase of the input\'s project'
+      )
+      return
+    end
+
+    if project.continuous?
+      errors.add(
+        :creation_phase,
+        :not_in_timeline_project,
+        message: 'The creation phase cannot be set for inputs in a continuous project'
+      )
+    end
```

Comment:
Shouldn't there be a `return` here, similar to line 235? Otherwise more than 1 error could come out and we agreed not to do that.
As discussed, I would move the continuous check to the top, below `return unless creation_phase`, because this check is more important than `creation_phase.project_id != project_id`.  After all, comparing project ids is not useful when a project is continuous.

Reply:
Good point, I forgot to add the return here!